server/utils/jsonApi.js

```diff
@@ -15,6 +15,26 @@ const getLargeImage = data => getImage(data, 'tile_large');
 
 const isNew = fromDate => differenceInDays(new Date(), new Date(fromDate)) <= 2;
 
+const cropTextWithEllipsis = (item, maxNumberOfChars) => {
+  if (!item || JSON.stringify(item) === '{}') {
+    throw new Error('An item object with the expected structure is required');
+  }
+
+  if (!maxNumberOfChars) {
+    return item;
+  }
```

Comment:
This masks an error in the code. This function should *never* be called without a number of chars. Unless there is a need to set a default that is? 🤔

Reply:
I agree the function 'should' never be called without maxNumberOfChars, this was coded in this way as a simple and none breaking method to handle that eventuality. The way I would change this would be to throw an error instead but I think its valid code to catch this in some form otherwise the function would just blow up.

Reply:
updated to use default value.